### ISSUE TITLE
올려주신 코드가 돌지를 않습니다. 모델 오류라고 뜨는데 어떻게 해야될까요?

### ISSUE THREAD

**Issue by cwoonb** [monologg/KoBigBird#7]
올려주신 코드를 우분투 20.4에서 구현을 하려고합니다. 
하지만 오류로 인해 3일 째 실행조차 못하고있습니다. 
finetuning을 하기위해 tpu를 제거 하라고 하셔서 쉣스크립트를 제거를 하였습니다. 
하지만 그렇게 하면 모델 오류가 발생을 합니다. 
모델을 찾을 수 없다는 오류 내용이 나타나는데 어떻게 해야되는지 알려주실 수 있을까요?
현재 새롭게 가상환경을 만들어서 돌려보실 때 잘돌아가나요?
- cuda 11.2
- python 3.7

erros:

**Comment by cwoonb** [comment]
> 올려주신 코드를 우분투 20.4에서 구현을 하려고합니다. 하지만 오류로 인해 3일 째 실행조차 못하고있습니다. finetuning을 하기위해 tpu를 제거 하라고 하셔서 쉣스크립트를 제거를 하였습니다. 하지만 그렇게 하면 모델 오류가 발생을 합니다. 모델을 찾을 수 없다는 오류 내용이 나타나는데 어떻게 해야되는지 알려주실 수 있을까요? 현재 새롭게 가상환경을 만들어서 돌려보실 때 잘돌아가나요?
> 
> * cuda 11.2
> * python 3.7
> 
>제발알려주세요 부탁드립니다.

**Comment by cwoonb** [comment]
#!/bin/bash

python3 run.py \
--task cls \
--dataset fake_news \
--do_train \
--do_eval_during_train \
--do_eval \
# --use_tpu \
--model_name_or_path monologg/kobigbird-bert-base \
--data_dir cache/fake_news_data \
--train_file mission2_train.csv \
--predict_file mission2_train.csv \
--max_seq_length 1024 \
--train_batch_size 4 \
--eval_batch_size 2 \
--learning_rate 3e-5 \
--gradient_accumulation_steps 2 \
--num_labels 2 \
--num_train_epochs 10
스크립트이고 

finetune/cache/fake_news(mission2_train.csv)와 말머리를 넣어놨습니다.

오류내용입니다.
usage: run.py [-h] --task TASK --dataset DATASET [--cache_dir CACHE_DIR]
              [--output_dir OUTPUT_DIR] [--do_train] [--do_eval_during_train]
              [--do_eval] [--use_tpu] [--threads THREADS] [--seed SEED]
              --model_name_or_path MODEL_NAME_OR_PATH [--data_dir DATA_DIR]
              [--train_file TRAIN_FILE] [--predict_file PREDICT_FILE]
              [--do_lower_case DO_LOWER_CASE]
              [--max_seq_length MAX_SEQ_LENGTH] [--weight_decay WEIGHT_DECAY]
              [--adam_epsilon ADAM_EPSILON] [--max_grad_norm MAX_GRAD_NORM]
              [--num_train_epochs NUM_TRAIN_EPOCHS]
              [--train_batch_size TRAIN_BATCH_SIZE]
              [--eval_batch_size EVAL_BATCH_SIZE]
              [--learning_rate LEARNING_RATE]
              [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS]
              [--warmup_proportion WARMUP_PROPORTION]
              [--num_labels NUM_LABELS]
run.py: error: the following arguments are required: --model_name_or_path
scripts/run_fake_news.sh: line 10: --model_name_or_path: command not found
